source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -164,7 +168,7 @@ void IDEASAppController::setupUserInterface()
 	ideasKETEKDetailedDetectorViewWithSave_->addCombinationPileUpPeakNameFilter(QRegExp("(Ka1|La1|Ma1)"));
 	mw_->addPane(ideasKETEKDetailedDetectorViewWithSave_, "XRF Detectors", "KETEK", ":/system-search.png");
 
-	ideas13ElementGeDetailedDetectorViewWithSave_ = new IDEASXRFDetailedDetectorViewWithSave(IDEASBeamline::ideas()->ge13Element());
+	g13ElementDetailedDetectorView *ideas13ElementGeDetailedDetectorViewWithSave_ = new g13ElementDetailedDetectorView(IDEASBeamline::ideas()->ge13Element());
```

Comment:
Same thing with this.  A couple other small pointers:
- Classes should all begin with capitals.
- Since this is the 13 element, it should be Ge (for germanium) not just g.
- Beamline specific classes should have their specific identifier (ie: IDEAS, SGM, etc.)